app/overrides/models/decidim/consultations/question.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Decidim::Consultations::Question.class_eval do
+  def total_delegates
+    total_votes = 0
+
+    authors = votes.select(:decidim_author_id).map(&:decidim_author_id)
+    granters_ids = Decidim::ActionDelegator::Delegation.where(granter: authors).map(&:granter_id).uniq!
```

Comment:
For performance, lets delegate all we can to the database, so better use something like
```suggestion
    granters_ids = Decidim::ActionDelegator::Delegation.select(:granter_id).where(granter: authors).group(:granter_id).pluck(:granter_id)
```

Reply:
Changes applied, thanks!